client/components/tinymce/plugins/mentions/suggestion-list.jsx

```diff
@@ -10,82 +10,72 @@ import PopoverMenu from 'components/popover/menu';
 import PopoverMenuItem from 'components/popover/menu-item';
 import Suggestion from './suggestion';
 
-const SuggestionList = React.createClass( {
-	displayName: 'SuggestionList',
+const getMatchingSuggestions = function( suggestions, query ) {
+	const matcher = new RegExp( '^' + query + '| ' + query, 'ig' ); // Start of string or preceded by a space.
```

Comment:
Do we only want to allow a space or would we rather allow space-characters? The ` ` could be replaced with the space character class `\s` if we want to be more general.